skimage/morphology/extrema.py

```diff
@@ -113,9 +113,7 @@ def h_maxima(image, h, selem=None):
 
     # Check for h value that is larger then range of the image. If this
     # is True then there are no h-maxima in the image.
-    im_min = np.min(image)
-    im_max = np.max(image)
-    if (h > (im_max - im_min)):
+    if (h > np.ptp(image)):
```

Comment:
No need for external parenthesis
```suggestion
    if h > np.ptp(image):
```